data/transaction.go

```diff
@@ -66,3 +77,33 @@ type SaveBlockData struct {
 	Scrs      map[string]smartContractResult.SmartContractResult `json:"scrs"`
 	LogEvents []Event                                            `json:"events"`
 }
+
+// ArgsSaveBlockData will contain all information that are needed to save block data
+type ArgsSaveBlockData struct {
+	HeaderHash             []byte
+	Body                   *block.Body
+	Header                 *block.HeaderV2
+	SignersIndexes         []uint64
+	NotarizedHeadersHashes []string
+	HeaderGasConsumption   outport.HeaderGasConsumption
+	TransactionsPool       *Pool
+	AlteredAccounts        map[string]*outport.AlteredAccount
+	NumberOfShards         uint32
+	IsImportDB             bool
+}
+
+// LogData holds the data needed for indexing logs and events
+type LogData struct {
+	LogHandler *transaction.Log
+	TxHash     string
+}
+
+// Pool will hold all types of transaction
```

Comment:
```suggestion
// Pool holds all types of transaction
```
to keep consistent with `LogData`

Reply:
done